TESTING.md

```diff
@@ -1,5 +1,5 @@
 # Testing 'oneview' cookbook
-The 'oneview' cookbook provides already unit testing strategy, besides having linting tools.
+The 'oneview' cookbook already provides a unit testing strategy, besides having and linting tools to ensure its code style.
```

Comment:
provides a unit testing strategy, **besides having**(?) and linting tools

Reply:
I knew that I forgot something...

![TEST](https://media.giphy.com/media/Rhhr8D5mKSX7O/giphy.gif)